blog/Getting Started with ANP.md

```diff
@@ -0,0 +1,270 @@
+
+- AdminNetworkPolicy rules should be interpreted as-is: pod-to-pod traffic only matches the
+  rule if all criteria listed in the rule are matched. That means src and dst of the packet
+  matches subject and ingress/egress peer selection, and in addition port/protocol also
+  matches, if those are specified in the rule. If so, `action` of the rule makes final verdict
+  on whether the packet is allowed and dropped, skipping any further policy rule evaluations.
+  Otherwise, traffic will continue to be matched against lower-precedence `AdminNetworkPolicy`
+  rules, K8s NetworkPolicies and finally, `BaselineAdminNetworkPolicy` rules, until all policy
+  rules are exhausted and no rule matches. At that time, Kubernetes' default networking
+  requirement will apply, which states that all pods can communicate with each other without
+  NAT.
+
+- AdminNetworkPolicy API does not change K8s NetworkPolicy's [isolation effect](
+  https://kubernetes.io/docs/concepts/services-networking/network-policies/#the-two-sorts-of-pod-isolation).
+  If a pod is selected by a NetworkPolicy with egress rules, any egress traffic from the pod,
+  if not explicitly allowed by an `AdminNetworkPolicy` rule or a NetworkPolicy rule, will be
+  dropped due to this isolation effect and thus will not be evaluated further against
+  `BaselineAdminNetworkPolicy` rules. The same applies for ingress direction traffic.
+
+With those characteristics of the AdminNetworkPolicy API in mind, let's take a look at how the
+two example policies secure cluster traffic as depicted below in a connection flow diagram. We
+have labeled a number of specific connections, for which we will take a deep dive into and see
+where in the policy enforcement hierarchy they matched AdminNetworkPolicy rules (if any). Note
+that currently there are no NetworkPolicy instances created in any namespace.
+
+![Diagram2](Diagram2.png)
+
+1. Traffic within the two sales tenant namespaces is allowed. Firstly, no `AdminNetworkPolicy`
+   rule matches this type of traffic. Both tenant=sales namespaces are indeed selected by the
+   "cluster-basic-allow" `AdminNetworkPolicy`, but the connection in question is neither going
+   to dns components nor from the monitoring namespace. Moving further down the hierarchy, no
+   `NetworkPolicy` can possibly be matched as well since there are none of them in the cluster
+   currently. Finally, the "default" `BaselineAdminNetworkPolicy` is evaluated, and this
+   traffic flow matches the first rule of the `BaselineAdminNetworkPolicy` egress section: for
+   packets sent by the `app=db` pod towards the `app=web` pod, the first egress rule is matched
+   from the perspective of namespace `s1`, since the rule dictates that for all namespaces any
+   egress traffic going to namespaces with the same `tenant` label should be given the green
+   light. The same logic applies for the reverse traffic.
```

Comment:
```suggestion
1. **Traffic between the sales tenant pods is allowed.**

  Firstly, no `AdminNetworkPolicy`
   rule matches this type of traffic. Both tenant=sales namespaces are indeed selected by the
   "cluster-basic-allow" `AdminNetworkPolicy`, but the connection in question is neither going
   to dns components nor from the monitoring namespace. Moving further down the hierarchy, no
   `NetworkPolicy` can possibly be matched as well since there are none of them in the cluster
   currently. Finally, the "default" `BaselineAdminNetworkPolicy` is evaluated, and this
   traffic flow matches the first rule of the `BaselineAdminNetworkPolicy` egress section: for
   packets sent by the `app=db` pod towards the `app=web` pod, the first egress rule is matched
   from the perspective of namespace `s1`, since the rule dictates that for all namespaces any
   egress traffic going to namespaces with the same `tenant` label should be given the green
   light. The same logic applies for the reverse traffic.
```

Maybe give each rule within the ANP a name as well? That might help simplify the explanation.  Also I think it'd be cool if the blub focused more so on the singe rule providing the desired network behavior. i.e This control is provided by the "allow-same-tenant" rule in the BANP. Rather than walking through each level for each number as it does get a bit confusing